### PR TITLE
feat: add detailed page for each solution

### LINKED ISSUE #12: Add Detailed Page for each Solution (3)
- [ ] c19.md

- [ ] voluntar.md

- [ ] ajut.md 

image

- [ ] Embed scribd report for voluntar.md
https://www.scribd.com/document/475134849/Suport-Grupurilor-Vulnerabile

- [ ] Embed scribd report for ajut.md
https://www.scribd.com/document/475136713/Implicarea-Societ%C4%83%C8%9Bii-Civile-in-ac%C8%9Biuni-Civice

https://www.figma.com/file/ro3AwDNZcYqWs86UHcEDtB/Code4.md-Website?node-id=113%3A1517

### PR DESCRIPTION
**Pull request by SpartakusMd**
- [x] Header block
- [x] Description block
- [x] Report block
- [x] Partners block
- [x] Other solutions block
- [x] Back to home button
- [ ] Progress block #25 

Closes #12

### CODE REVIEW

File: data/solutions-data.tsx [hunk 1]
```diff
@@ -7,25 +7,81 @@ import moldovaSvg from '../images/moldova.svg'
...
+      '<p class="mb-2"><strong>Problema:</strong> Un număr considerabil de pesoane aflate în situații de criză (nu au acces la produse alimentare, de igienă și acces la îngrijire medicală, etc.) nu beneficiază de suport.</p>' +
+      '<p class="mb-2"><strong>Soluția:</strong> Platformă online care conectează persoanele aflate în situații de criză și societatea civilă care ajută cu produse alimentare, de igienă și transport.</p>' +
+      '<ul class="list-disc list-inside pl-5">' +
+      '<li>Toate solicitările telefonice a persoanelor vulnerabile sunt înregistrate pe această platformă</li>' +
+      '<li>Orice voluntar care este dispus să ajute cu livrarea pachetelor și oferirii de transport poate să se înregistreze pe platformă</li>' +
+      '<li>Voluntarul primește acces la platformă și poate prelua solicitări</li>' +
+      '<li>Progresul fiecărei sarcini și statutul solicitărilor pot fi ușor gestionate și monitorizate</li>' +
+      '</ul>',
```
Comment (SpartakusMd) [inline data/solutions-data.tsx:43]: Used HTML as we'll receive HTML from the CMS :)

File: blocks/back-button.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { useRouter } from 'next/router'
+
+export function BackButton() {
```
Comment (unrevised6419) [inline blocks/back-button.tsx]: Move this component in `components` folders. In `blocks` we keep only big sections

File: data/solutions-data.tsx [hunk 3]
```diff
@@ -7,25 +7,86 @@ import moldovaSvg from '../images/moldova.svg'
...
     image: girl,
     icon: moldovaSvg,
+    status: 'finisat',
```
Comment (unrevised6419) [inline data/solutions-data.tsx]: Is not finished yet :slightly_smiling_face:

Comment (SpartakusMd) [reply data/solutions-data.tsx]: Took from the mockups :) Will remove it.

**Review by unrevised6419 — APPROVED** [review]
:rocket:

**Pull request merged**